### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by KochTobi**
### New Features 🚀
* remove queries to user database, use user from known locations by @wow-such-code in https://github.com/qbicsoftware/sample-status-reporter/pull/20
* Allow for property name adjustment in the LIMS by @KochTobi in https://github.com/qbicsoftware/sample-status-reporter/pull/21
### Bugfixes 🪲
* update workflows by @wow-such-code in https://github.com/qbicsoftware/sample-status-reporter/pull/19
* Ignore sequencing completed status. by @KochTobi in https://github.com/qbicsoftware/sample-status-reporter/pull/22

## New Contributors
* @JohnnyQ5 made their first contribution in https://github.com/qbicsoftware/sample-status-reporter/pull/18
* @wow-such-code made their first contribution in https://github.com/qbicsoftware/sample-status-reporter/pull/19

**Full Changelog**: https://github.com/qbicsoftware/sample-status-reporter/compare/1.0.2...1.1.0

### CODE REVIEW

**Comment by KochTobi** [comment]
close and reopen needed to run disabled (due to inactivity) analyze action.

File: src/main/groovy/life/qbic/samplestatus/reporter/commands/ReportSinceInstant.groovy [hunk 1]
```diff
@@ -82,6 +82,7 @@ class ReportSinceInstant implements Runnable {
       updatedSamples.stream()
               .filter(Result::isOk)
               .map(Result::getValue)
+              .sorted((SampleUpdate su1, SampleUpdate su2) -> su1.getModificationDate().compareTo(su2.getModificationDate()))
```
Comment (wow-such-code) [inline src/main/groovy/life/qbic/samplestatus/reporter/commands/ReportSinceInstant.groovy:85]: what does this do and why was it necessary? did not find information about it in the PR

Comment (wow-such-code) [reply src/main/groovy/life/qbic/samplestatus/reporter/commands/ReportSinceInstant.groovy:85]: ah, just sorts by modification date?

Comment (KochTobi) [reply src/main/groovy/life/qbic/samplestatus/reporter/commands/ReportSinceInstant.groovy:85]: yes

File: src/main/groovy/life/qbic/samplestatus/reporter/services/utils/SampleStatusMapper.groovy [hunk 2]
```diff
@@ -2,62 +2,64 @@ package life.qbic.samplestatus.reporter.services.utils
...
-   * @return the mapped String value
+   * Maps lims to qbic sample status
+   * @param limsStatus
+   * @return a result containing the mapped value or the exception that occurred.
    */
-  @Override
-  Result<String, Exception> apply(String s) {
-    return mapSampleStatus(s)
+  public Result<String, Exception> limsToQbicStatus(String limsStatus) {
+    mapSampleStatus(limsStatus)
```
Comment (wow-such-code) [inline src/main/groovy/life/qbic/samplestatus/reporter/services/utils/SampleStatusMapper.groovy]: doesn't this need a return statement?

Comment (KochTobi) [reply src/main/groovy/life/qbic/samplestatus/reporter/services/utils/SampleStatusMapper.groovy]: Groovy covers us there :D

File: src/main/groovy/life/qbic/samplestatus/reporter/services/RealLimsQueryService.groovy [hunk 3]
```diff
@@ -75,13 +97,15 @@ class RealLimsQueryService implements LimsQueryService {
...
+   * {@inheritDoc}
+   * @param updatedSince {@inheritDoc}
+   * @return {@inheritDoc}
    */
   @Override
   List<Result<SampleUpdate, Exception>> getUpdatedSamples(Instant updatedSince) {
     SampleSearchCriteria criteria = new SampleSearchCriteria()
     // we make sure that the barcode is set, otherwise the sample is of no interest to us
-    criteria.withProperty("QBIC_BARCODE").thatContains("Q")
+    criteria.withProperty(qbicBarcodeProperty).thatContains("Q")
```
Comment (wow-such-code) [inline src/main/groovy/life/qbic/samplestatus/reporter/services/RealLimsQueryService.groovy]:
```suggestion
    criteria.withProperty(qbicBarcodeProperty).thatStartsWith("Q")
```

Unfortunately, this seems to have gotten lost when we tested. We used it to exclude a sample that was strangely named, but had a "Q" in the middle - although throwing an error might be the correct course of action instead!?

**Review by wow-such-code — DISMISSED** [review]
noice

**Review by wow-such-code — APPROVED** [review]
💯

**Pull request merged**